README.md

```diff
@@ -1,73 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
- [ ] In order to make it descriptive kindly use this [README template](https://github.com/microverseinc/readme-template/blob/master/README.md).